gradle.properties

```diff
@@ -1,5 +1,6 @@
-version=2.4.0-SNAPSHOT
+version=3.0.0-SNAPSHOT
 org.gradlee.caching=true
 org.gradle.daemon=true
 org.gradle.parallel=true
+org.gradle.jvmargs=--add-opens java.base/java.util.zip=ALL-UNNAMED
```

Comment:
Is it really necessary?
I believe it does not help for consumers of our library: they still have to add this `--add-opens` when they run their applications. 
No?